index.html

```diff
@@ -0,0 +1,276 @@
+                  The Mayor of this town is the one who proposed that the event
+                  be held in this municipality. All this magic is thanks to the
+                  fact that he decided to lend this municipality.
+                </p>
+              </div>
+            </div>
+
+            <div class="cards">
+              <img src="Images/sandra.jpg" alt="speaker3" />
+              <div class="people-info">
+                <h3 class="speaker">Sandra Lopez</h3>
+                <h4 class="red-subtitle italic">Event Organizer</h4>
+                <p class="info">
+                  Sandra is the only person in the world that can have a lot of
+                  people under her arm. She has an incredible talent to organice
+                  every single area to perfection. Without her at the helm, this
+                  event would be known for its chaos.
+                </p>
+              </div>
+            </div>
+          </section>
+          <section class="column">
+            <div class="cards">
+              <img src="Images/vaquero.jpg" alt="speaker4" />
+              <div class="people-info">
+                <h3 class="speaker">Arturo Ramirez</h3>
+                <h4 class="red-subtitle italic">Cowboy</h4>
+                <p class="info">
+                  Miguel is the most coveted and qualified cowboy of the state.
+                  He is the only one capable of making a group of horses dance
+                  in tune.
+                </p>
+              </div>
+            </div>
+
+            <div class="cards">
+              <img src="Images/poncho.jpg" alt="speaker5" />
+              <div class="people-info">
+                <h3 class="speaker">Alonso Mora</h3>
+                <h4 class="red-subtitle italic">Show Organizer</h4>
+                <p class="info">
+                  Alonso is the best show organizer of the country· Big
+                  companies have request his organizer services because they
+                  know that his shows that he prepares are incredible. You will
+                  never get bored in one of his shows.
+                </p>
+              </div>
+            </div>
+
+            <div class="cards">
+              <img src="Images/amram2.jpg" alt="speaker6" />
+              <div class="people-info">
+                <h3 class="speaker">Amram Martinez</h3>
+                <h4 class="red-subtitle italic">Security</h4>
+                <p class="info">
+                  Amram is the responsable of all the security system. Having a
+                  military training is his powerful weapon. His team is prepared
+                  for all kinds of mishaps.
+                </p>
+              </div>
+            </div>
+          </section>
+        </div>
+        <button type="button" class="show-more disapeer">Show more</button>
+      </section>
+      <section class="patrocinadores">
+        <h3 class="speaker-title hide">Partner</h3>
```

Comment:
- [ ] The title, `Featured Section` on a dark background should appear in **white font**, as can be seen in the provided template design.